Library/Source/RuntimeImpl.cpp

```diff
@@ -36,12 +39,12 @@ namespace babylon
             .Get(JS_WINDOW_NAME).ToObject());
     }
 
-    RuntimeImpl::RuntimeImpl(void* nativeWindowPtr, const std::string& rootUrl)
+    RuntimeImpl::RuntimeImpl(void* nativeWindowPtr, const std::string& rootUrl, LogCallback&& logCallback)
         : m_nativeWindowPtr{ nativeWindowPtr }
         , m_thread{ [this] { ThreadProcedure(); } }
         , m_rootUrl{ rootUrl }
-    {
-    }
+        , m_logCallback{ logCallback }
+    {}
```

Comment:
nit: I don't like `{}`. I prefer `{` and `}` on separate lines. multiple places.